Sources/iOS/Library/CollectionAdapter.swift

```
@@ -422,7 +401,7 @@ extension CollectionAdapter : UICollectionViewDataSource {
   public func collectionView(collectionView: UICollectionView, cellForItemAtIndexPath indexPath: NSIndexPath) -> UICollectionViewCell {
     spot.component.items[indexPath.item].index = indexPath.item
 
-    let reuseIdentifier = spot.reuseIdentifierForItem(indexPath)
+    let reuseIdentifier = spot.identifier(indexPath) ?? ""
```

Should we default to the `Component` default instead of returning an empty string, I'm pretty sure that will crash 💥